L1Trigger/TrackFindingTracklet/src/TrackletLUT.cc

```diff
@@ -478,6 +478,43 @@ void TrackletLUT::initProjectionBend(double k_phider,
   writeTable();
 }
 
+void TrackletLUT::initProjectionDiskRadius(int nrbits) {
+  double roffset = 3.0;
```

Comment:
Add a comment explaining what roffset represents physically

Reply:
@aryd what exactly does this correspond to?

Reply:
I have added a comment to explain this.